apps/frontend/src/app/pages/company/interview-gpt/interview-gpt.component.html

```diff
@@ -12,10 +12,20 @@ <h3 class="font-sans font-medium text-2xl text-base-100">
     </div>
   </div>
   <p class="font-mono font-bold text-sm text-base-100 pb-1">Set questions</p>
```

Comment:
Lets change this text to something more descritive (like "Set a minimum of X questions") and then we can merge